README.md

```diff
@@ -46,7 +46,7 @@ and [Search API Solr](https://www.drupal.org/project/search_api_solr) on [Panthe
 Search API Solr provides the ability to connect to any Solr server by providing numerous configuration options.
 This module automatically sets the Solr connection options by extending the plugin from Search API Solr.
 The module also changes its connection information based on different Pantheon environments and each
-Pantheon Environment has it's own [SOLR CORE](). Doing so eliminates the need to do extra work setting up Solr servers for each environment.
+Pantheon Environment has it's own [SOLR CORE](#). Doing so eliminates the need to do extra work setting up Solr servers for each environment.
```

Comment:
Should SOLR CORE be in all caps?

Reply:
I leave that to @stovak but it could be: 

```suggestion
Pantheon Environment has it's own [Solr Core](#). Doing so eliminates the need to do extra work setting up Solr servers for each environment.
```

Without a link there, it was hard for me to know for sure with no source to check.